buildSrc/src/main/java/org/elasticsearch/gradle/LazyString.java

```diff
@@ -17,15 +17,25 @@
  * under the License.
  */
 
-package org.elasticsearch.packaging.test;
+package org.elasticsearch.gradle;
 
-import org.elasticsearch.packaging.util.Distribution;
+import java.util.function.Supplier;
 
-public class DefaultNoJdkDebBasicTests extends PackageTestCase {
+/**
+ * An functional interface to allow a String supplier to be resolved when calling toString().
+ *
+ * This can be used in many places where a string expression should not be evaluated until runtime.
+ */
+public class LazyString {
```

Comment:
This class doesn't seem to be used anywhere.

Reply:
Removed. This was leftover from an avenue I explored.